### PR TITLE
Fix autocomplete

### LINKED ISSUE #458: Fix AutoComplete compatibility with callbacks
## Description
 Dash callbacks should be able to update from a change in Search Text in the Autocomplete Component, but that is not currently possible

### PR DESCRIPTION
**Pull request by dwkaminsky**
## Description
Updates Autocomplete Component to allow a change in search text to trigger a Dash Callback

## Motivation and Context
Closes #458 

## Screenshots (if appropriate):
captured (11)

### CODE REVIEW

File: usage.py [hunk 1]
```diff
@@ -65,6 +65,7 @@
                         dict(label='option 3', value='3'),
                     ],
                     valueSelected='1',
+                    row=True,
```
Comment (coralvanda) [inline usage.py:68]: This works, but it takes up a lot of room. Let's keep it as a column of buttons to fit the layout.

**Comment by coralvanda** [comment]
@dwkaminsky I had some comments from a previous review, please take a look at those, too.

File: src/lib/components/AutoComplete.react.js [hunk 2]
```diff
@@ -97,10 +97,6 @@ type Props = {
   openOnFocus?: boolean,
   /** Text being input to auto complete */
   searchText?: string,
-  /** Value in the dataSource found by using searchText
-   * NOTE exactMatch must be true for this to work
-   */
-  searchValue?: any,
```
[the remark this answers is not in the document]
Comment (dwkaminsky) [reply src/lib/components/AutoComplete.react.js:103]: Yes, that is correct

**Comment by dwkaminsky** [comment]
@coralvanda Sorry I missed those. Ready now

File: usage.py [hunk 3]
```diff
@@ -127,8 +127,14 @@
                                     {'label': 'Chicago, IL', 'value': 'Chicago'},
                                     {'label': 'Detroit, MI', 'value': 'Detroit'},
                                     {'label': 'Los Angeles, CA', 'value': 'Los Angeles'}],
-                        ),
-                    html.P(id='autocomplete-output')
+                        exactMatch=True,
```
Comment (coralvanda) [inline usage.py:130]: Let's completely remove this prop from here, and from the component.

**Pull request merged**